binderhub/build.py

```diff
@@ -176,11 +176,13 @@ def __init__(
     def get_r2d_cmd_options(self):
         """Get options/flags for repo2docker"""
         r2d_options = [
-            '--ref', self.ref,
-            '--image', self.image_name,
-            '--no-clean', '--no-run', '--json-logs',
-            '--user-name', 'jovyan',
-            '--user-id', '1000',
+            f"--ref={self.ref}",
+            f"--image={self.image_name}",
+            "--no-clean",
+            "--no-run",
+            "--json-logs",
+            "--user-name=jovyan",
+            "--user-id=1000",
```

Comment:
Note I've tested repo2docker the to function with `--arg=value` and not only with `--arg value` as I know not all CLI may support that, but repo2docker does.